src/index.js

```diff
@@ -63,32 +50,29 @@ module.exports = function module (moduleOptions) {
       staticRoutes = staticRoutes.filter(route => minimatch.match(route))
     })
 
-    if (this.options.dev || options.generate) {
+    if (this.options.dev) {
       // Create a cache for routes
       cache = createCache(staticRoutes, options)
-    }
-
-    if (!this.options.dev) {
-      // TODO on build process only
+    } else {
       // Save static routes
       fs.ensureDirSync(path.resolve(this.options.buildDir, 'dist'))
       fs.writeJsonSync(jsonStaticRoutesPath, staticRoutes)
+    }
+  })
 
-      // TODO on generate process only and not on build process
-      if (options.generate) {
```

Comment:
Last thing I promise! 😄 

A notice for the user that `generate` isn't needed anymore

Reply:
@manniL great review!
Added notice with `consola.warn`